src/emulator/controller.ts

```diff
@@ -293,29 +303,33 @@ export async function startAll(options: any, noGui: boolean = false): Promise<vo
       if (fs.existsSync(rules)) {
         args.rules = rules;
       } else {
-        utils.logLabeledWarning(
-          "firestore",
+        firestoreLogger.logLabeled(
+          "WARN",
+          Emulators.FIRESTORE,
```

Comment:
nit: this is the only place you're using the `Emulators` constant as a label, in other places it's just the string.  I'd suggest sticking with the string here (plus we're gonna remove that arg soon)

Reply:
I guess it's here through the end of the file

Reply:
yeah I'll replace it